src/routes/profile.tsx

```diff
@@ -138,7 +135,53 @@ function Profile() {
   const [reviewsError, setReviewsError] = useState<string | null>(null);
   const [reviewsPage, setReviewsPage] = useState(0);
   const [hasMoreReviews, setHasMoreReviews] = useState(false);
+  const [userData, setUserData] = useState<User | null>(null);
+  const [userLoading, setUserLoading] = useState(true);
+  const [userError, setUserError] = useState<string | null>(null);
+  
   const reviewsLimit = 5;
+  
+  // Assuming we have a user ID - in a real app, this would come from auth context or similar
+  const userId = "uuid-user3"; // Replace with actual user ID or get from auth context
+
+  // Fetch user data
+  useEffect(() => {
+    fetchUserData();
+  }, []);
+
+  async function fetchUserData() {
+    setUserLoading(true);
+    setUserError(null);
+
+    try {
+      // const gatewayUrl = import.meta.env.VITE_GATEWAY_URL || "http://localhost:61931";
+      const res = await fetch(`http://localhost:61931/get?id=${userId}`);
```

Comment:
```suggestion
      const res = await fetch(`http://localhost:61931/get?id=${userId}`);
```
Use environment variable instead of localhost